conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md

```diff
@@ -44,28 +44,32 @@ in place by a project (for reports made to a project) or the CPC (in the
 case of reports relating to spaces within the scope of the foundation versus a project).
 In exceptional cases where a reporter wishes to challenge the response from the CPC or
 project, or does not feel comfortable reporting through the standard processes, an escalation
-report can be made to coc-escalation@openjsf.org.
+report can be made to report@openjsf.org.
 
 All escalations will be reviewed and investigated by the Code of Conduct Panel (CoCP)
 and will result in a response that is deemed necessary and appropriate to the circumstances.
 The Code of Conduct panel is obligated to maintain confidentiality with regard
 to the reporter of an escalation.
 
-### Code of Conduct panel
+### Code of Conduct Panel
```

Comment:
```suggestion
### OpenJS Foundation Code of Conduct team
```

Reply:
Wouldn't "moderation team" like in Node be clearer? (not suggesting that necessarily just asking)

Reply:
That team won't really be doing proactive moderation. It's more reactive (responding to reports), because it sits at the foundational level, and not at the project level.

Reply:
Makes sense I agree.